packages/components/src/number-control/index.js

```diff
@@ -72,7 +76,7 @@ export function NumberControl(
 			const enableShift = event.shiftKey && isShiftStepEnabled;
 
 			const incrementalValue = enableShift
-				? parseFloat( shiftStep )
+				? parseFloat( shiftStep ) * parseFloat( step )
```

Comment:
Improvement: The modifier now accounts for shiftStep **and** step (used as a multiplier)

Reply:
So, if I'm understanding this correctly, this change makes it so that:

- user doesn't hold shift => values will increment by `step` (ex: 5) => 5 by 5
- user holds shift => values will increment by `shiftStep` (ex: 10) * `step` (ex: 5) => 50 by 50

The actual variation that happens when the user holds down shift is always tied to the `step`, and `shiftStep` works as an accelerator. It makes sense to me, I can't think of any cases where using two unrelated values would make sense (like `step` = 3 and `shiftStep` = 7).

However, I'm wondering if this is a breaking change. It seems to me like it is, given that consumers of this API will have to change the value they pass to `shiftStep` to get the same results when this lands. I guess then this would require the appropriate changelog note.

Reply:
> So, if I'm understanding this correctly, this change makes it so that:

@nosolosw Yes! You are correct. This became (more) evident to me when I was working with `0.01` step values. I expected the `shift` jump needed to be relative to the step value.

>  I guess then this would require the appropriate changelog note.

Happy to provide that!

Reply:
In addition to the changelog, this component's README file also needs updating to explain the new relationship between step/shiftStep.